apps/ui/src/views/Space/Editor.vue

```diff
@@ -47,7 +47,8 @@ const {
   reset
 } = useWalletConnectTransaction();
 const proposalsStore = useProposalsStore();
-const { votingPower, fetch: fetchVotingPower } = useVotingPower();
+const { get: getProposingPower, fetch: fetchProposingPower } =
```

Comment:
```suggestion
const { get: getPropositionPower, fetch: fetchPropositionPower } =
```

I think we should stick with `proposition` as a constant term and use it as-is everywhere.